gnd/src/main/res/navigation/nav_graph.xml

```diff
@@ -67,8 +67,22 @@
         android:defaultValue="NEW_RECORD"
         android:name="recordId"/>
     </action>
+    <action
+        android:id="@+id/showBasemapSelector"
+        app:destination="@id/basemap_selector_fragment"
+        />
   </fragment>
 
+  <fragment
+    android:id="@+id/basemap_selector_fragment"
+    android:label="Offline Maps"
```

Comment:
Please refer to string resource here instead.

Reply:
Doh! Don't know how I missed that, thanks!